docs/setup-guide/cluster_setup.md

```diff
@@ -0,0 +1,433 @@
+     width="98%"  
+     align="center"/>
+     
+### 2.1.2 Another window run following line, to check machine status and mission deployment status (Step 5)
+```
+kubectl get crd
+kubectl get edgecluster
+kubectl get mission
+```
+
+##  2.2		In machine B. (Notes: If we have C machine, we need also run "cloudcore" in machine B.)
+### 2.2.1 Run edgecore in machine B (Step 2)
+- following command line only run one time.
+```
+chmod 777 /root/go/src/github.com/kubeedge/_output/local/bin/kubectl/vanilla/kubectl
+```
+```
+export KUBECONFIG=/etc/kubernetes/admin.conf
+_output/local/bin/edgecore --edgecluster
+```
+- Notes: If you first time to run above command, please run the following command. only run one time.
+```
+chmod 777 /root/go/src/github.com/kubeedge/_output/local/bin/kubectl/vanilla/kubectl
+```
+
+### 2.2.2 Run cloudcore in machine B (if you have machine C) (Step 3)
+```
+export KUBECONFIG=/etc/kubernetes/admin.conf
+_output/local/bin/cloudcore
+```
+Result:
+<img src="images/Running_B_edgecoreAndcloudcore.png" 
+     width="98%"  
+     align="center"/>
+
+##  2.3		In machine C. (only run edgecore)
+### 2.3.1 Run edgecore in machine C (Step 4)
+- following command line only run one time.
+```
+chmod 777 /root/go/src/github.com/kubeedge/_output/local/bin/kubectl/vanilla/kubectl
+```
+```
+export KUBECONFIG=/etc/kubernetes/admin.conf
+_output/local/bin/edgecore --edgecluster
+```
+- Notes: If you first time to run above command, please run the following command. only run one time.
+```
+chmod 777 /root/go/src/github.com/kubeedge/_output/local/bin/kubectl/vanilla/kubectl
+```
+
+Result:
+<img src="images/Running_C_edgecore.png" 
+     width="98%"  
+     align="center"/>
+
+# 3	Deployment Mission to Machine B, C
+## 3.1	Waiting machine A, B, C running, we can start 3.2
+## 3.2  In machine A, do following command in Second Command Window.
+
+```
+kubectl apply -f tests/edgecluster/data/missions/deployment-to-all.yaml
+```
+you will see one line have the message **Mission deployment-to-all is created**. detail see the result
+<BR>Result:
+<img src="images/Deployment_mission_status.png" 
+     width="98%"  
+     align="center"/>
+
+## 3.3 Run following command verify mission created status(in second window).
+```
+kubectl get edgeclusters
+kubectl get missions
+```
+Result:
+<img src="images/Deployment_3_2_status.png" 
+     width="98%"  
+     align="center"/>
+     
+## 3.4 After you see mission, you can test delete mission. by using following command
+```
+kubectl delete mission deployment-to-all
+```
+## 3.5 Run following command verify mission deteted status(in second window).
+```
+kubectl get edgeclusters
+kubectl get missions
+```
+OR
+```
+watch kubectl get edgeclusters
+watch kubectl get missions
+```
+	
+Result:
+<img src="images/Deployment_3_5_status.png" 
+     width="98%"  
+     align="center"/>
+
+
+# 4	Tips & Tricks
+## 4.1  When you finished run the cloudcore and edgecore. Please use "Ctrl + c" exit running.
```

Comment:
the format of this section doesn't look right

![image](https://user-images.githubusercontent.com/252020/146838379-0db30c50-6107-4400-b367-31b3fcbb7940.png)

Reply:
Corrected format